projects/1.19/assets/hellish-materials/hellish-materials/lang/zh_cn.json

```
@@ -0,0 +1,56 @@
+{
+    "item.hellish-materials.luss_dust": "拉斯粉",
+    "item.hellish-materials.luss_ingot": "拉斯锭",
+    "item.hellish-materials.luss_axe": "拉斯斧",
+    "item.hellish-materials.luss_hoe": "拉斯锄",
+    "item.hellish-materials.luss_pickaxe": "拉斯镐",
+    "item.hellish-materials.luss_shovel": "拉斯锹",
+    "item.hellish-materials.luss_sword": "拉斯剑",
+    "block.hellish-materials.luss_ore": "拉斯矿石",
+    "block.hellish-materials.luss_block": "拉斯块",
+    "item.hellish-materials.vuld_helm": "沃伍德头盔",
+    "item.hellish-materials.vuld_chestplate": "沃伍德胸甲",
+    "item.hellish-materials.vuld_leggings": "沃伍德护腿",
+    "item.hellish-materials.vuld_boots": "沃伍德靴子",
+    "item.hellish-materials.vuld": "沃伍德",
+    "item.hellish-materials.vuld_drop": "沃伍德液滴",
+    "item.hellish-materials.vuld_axe": "沃伍德斧",
+    "item.hellish-materials.vuld_hoe": "沃伍德锄",
+    "item.hellish-materials.vuld_pickaxe": "沃伍德镐",
+    "item.hellish-materials.vuld_shovel": "沃伍德锹",
+    "item.hellish-materials.vuld_sword": "沃伍德剑",
+    "item.hellish-materials.flask_of_vaporous_vuld": "气态沃伍德瓶",
+    "block.hellish-materials.vuld_ore": "染污玄武岩",
+    "block.hellish-materials.corrupted_bone": "腐化骨块",
+    "block.hellish-materials.vaporous_vuld": "气态沃伍德",
+    "item.hellish-materials.batet_helm": "巴忒特头盔",
+    "item.hellish-materials.batet_chestplate": "巴忒特胸甲",
+    "item.hellish-materials.batet_leggings": "巴忒特护腿",
+    "item.hellish-materials.batet_boots": "巴忒特靴子",
+    "item.hellish-materials.batet_gem": "巴忒特",
+    "item.hellish-materials.batet_fragment": "巴忒特碎片",
+    "item.hellish-materials.batet_axe": "巴忒特斧",
+    "item.hellish-materials.batet_hoe": "巴忒特锄",
+    "item.hellish-materials.batet_pickaxe": "巴忒特镐",
+    "item.hellish-materials.batet_shovel": "巴忒特锹",
+    "item.hellish-materials.batet_sword": "巴忒特剑",
+    "block.hellish-materials.batet_ore": "巴忒特矿石",
+    "block.hellish-materials.batet_block": "巴忒特块",
+    "effect.hellish-materials.bleeding": "出血",
+    "effect.hellish-materials.guts": "损毁",
+    "item.hellish-materials.moratorium": "缓期",
+    "item.hellish-materials.luckstone": "幸运石",
+    "item.hellish-materials.luss_dice": "拉斯骰子",
+    "item.hellish-materials.exploding_dice": "爆炸骰子",
+    "death.attack.hmBloodDebt": "%1$s被往昔吞噬了",
+    "death.attack.hmBloodDebt.player": "在和%2$s战斗时%1$s被往昔吞噬了",

```

```suggestion
    "death.attack.hmBloodDebt.player": "%1$s在与%2$s战斗时被往昔吞噬了",
```
https://minecraft.fandom.com/zh/wiki/%E6%AD%BB%E4%BA%A1%E6%B6%88%E6%81%AF